keyboards/system76/launch_1/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Allow flashing with USBasp
+AVRDUDE_MCU = $(MCU)
+
```

Comment:
```suggestion
```

Reply:
This definition is needed for the `usbasp` target which flashes the AVR through ISP. It is not set to any default for the AVR platform.

Reply:
ISP flashing in this way is not really supported anyway... just link to the ISP flashing guide.

Reply:
I see a target in the platform makefile using this variable. You're suggesting that functionality is deprecated and/or should never be used?

Reply:
it's an atmega32u4, does it not have the stock atmel-dfu bootloader?

Reply:
@jecassis that target is for flashing boards which use the USBaspLoader bootloader, which emulates a USBasp device. There is no way to change fuse settings using the QMK makefile system, so it is not suitable as a replacement for following the ISP flashing guide.

Reply:
Appreciate the explanation, @fauxpark. I'll remove it as suggested.

@drashna If you look at the System76 fork, they modify the AVR platform makefile to work with the `production` target by concatenating the stock Atmel-DFU bootloader (bypassing the LUFA DFU bootloader compile). So, I assume that is what they're using. As a side note, I removed that change from this PR as I didn't want to touch core files.